keel-core/keel-core.gradle

```diff
@@ -24,8 +24,6 @@ dependencies {
   api "com.fasterxml.jackson.datatype:jackson-datatype-jsr310"
   api "de.huxhorn.sulky:de.huxhorn.sulky.ulid:${spinnaker.version("ulid")}"
 
-  implementation "com.google.guava:guava:${spinnaker.version("guava")}"
```

Comment:
Version in spinnaker-dependencies seems to be invalid but regardless, it turns out we don't need this (probably cruft from gRPC days).